lib/test-endpoint.sh

```diff
@@ -2,13 +2,26 @@
 
 unset DD_LOGS_VALID_ENDPOINT
 DATADOG_DIR="${DATADOG_DIR:-/home/vcap/app/datadog}"
+DD_US_API_SITE="https://api.datadoghq.eu/api/"
+DD_EU_API_SITE="https://api.datadoghq.us/api/"
```

Comment:
it's the other way around :)

Reply:
`.com` for US? I don't think we use `.us` anywhere.